2020/06.md

```diff
@@ -96,6 +96,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 0     | 30m     | [Intl Enumeration API](https://github.com/FrankYFTang/proposal-intl-enumeration) for Stage 1| Frank Yung-Fong Tang |
     |   | 0     | 30m     | [Async Context](https://github.com/legendecas/proposal-async-context) for Stage 1 (slides WIP) | Chengzhong Wu |
     |   | 0     | 45m     | [Subclassing support in built-in methods delenda est](https://github.com/syg/proposal-rm-builtin-subclassing) for Stage 1 [slides](https://docs.google.com/presentation/d/1vJeJFueDwrj8ebXFdGsEO1J_Q-DzfU01dLEGVd26A9o/edit#slide=id.p) | Yulia Startsev |
+    |   | 0     | 30m     | [Deep Path Properties in Record Literals](https://github.com/rickbutton/proposal-deep-path-properties-for-record) for Stage 1 [slides](https://button.dev/talks/deep-path-properties-tc39-june-2020.pdf) | Rick Button |
```

Comment:
Please respect the proposal topic sort order. This should be listed at the end of the stage 0, 30 minute timebox items.